### ISSUE TITLE
Change default keybinds that conflict with Atom core

### ISSUE THREAD

**Issue by db47h** [skandasoft/navigate#11]
A default install of Atom uses the "almost" standard F2 to navigate bookmarks and F3 for find-next. Overriding core keybinds may confuse users.

**Comment by skandasoft** [comment]
you can change the keymap.cson file to update the f2/f3 combination. In fact there are other keys like ctr+f1/f2/f3/f4 --> configured to different things like google help/doc help/stack overflow etc...If you go into lib/config.coffee(in the navigate package) you can change them to the site you like..

**Comment by db47h** [comment]
I gave F2 and F3 only as an example of keys mapped in core and overridden by navigate. If the user changes the keybinds in lib/config.coffee, won't his changes be overwritten by the next update of the package?

Currently, the only viable option I have found is to install yet another package to disable navigate's keybinds so I can remap them reliably in my keymap.cson.

Overriding core keybinds in a package IS a pain for end users.
